android/app/src/main/kotlin/com/rtirl/chat/NotificationService.kt

```diff
@@ -0,0 +1,124 @@
+package com.rtirl.chat
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.app.Service
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import android.os.IBinder
+import android.speech.tts.TextToSpeech
+import android.util.Log
+import androidx.core.app.NotificationCompat
+import com.rtirl.chat.R
+
+class NotificationService : Service() {
```

Comment:
Does the flutter notification lib we use support a button? I'm surprised this has to be done manually.

Reply:
Not quite sure how it works on android and I might have messed this up but this was my attempt to ditch awesome_notifications